### PR TITLE
Allow openqa-bootstrap to clone job in non-systemd environments

### PR DESCRIPTION
**Pull request by baierjan**
Run openqa-worker on background to allow script to continue with openqa-clone-job invocation, then wait for the worker process.

Reference: https://progress.opensuse.org/issues/164595

### CODE REVIEW

File: script/openqa-bootstrap [hunk 1]
```diff
@@ -182,3 +182,5 @@ start-worker
 if [[ $# -ne 0 ]]; then
     openqa-clone-job "$@"
 fi
+
+[[ -n $running_systemd ]] || wait
```
Comment (nicksinger) [inline script/openqa-bootstrap:186]: we know from the current problem that `worker` will never stop by itself - wouldn't that mean `openqa-bootstrap` would again never finish but this time waiting at the end?

Comment (baierjan) [reply script/openqa-bootstrap:186]: Yes and that is intended, see https://progress.opensuse.org/issues/164595?#note-5

TLDR: entrypoint for openqa-single-instance container

Comment (nicksinger) [reply script/openqa-bootstrap:186]: ah, that explains it. Would you mind adding a short comment to that line explaining it? Maybe even referencing the comment?

Comment (nicksinger) [reply script/openqa-bootstrap:188]: awesome, thanks!

**Review by nicksinger — APPROVED** [review]

**Review by Martchus — APPROVED** [review]
